### PR TITLE
Copy snapshot url link in snapshot header

### PR DESCRIPTION
**Pull request by PixelJanitor**

### CODE REVIEW

File: app/components/snapshot-viewer.js [hunk 1]
```diff
@@ -73,6 +72,7 @@ export default Component.extend({
     selectChild() {
       this.get('setAsSelected').call(this);
       this.get('selectChild')(this);
+      this.get('flashMessages').success('Snapshot URL was copied to your clipboard');
```
Comment (cadeParade) [inline app/components/snapshot-viewer.js]: Get rid of this line to make the tests pass

File: app/components/snapshot-viewer.js [hunk 2]
```diff
@@ -3,9 +3,8 @@ import {computed} from '@ember/object';
 import Component from '@ember/component';
 
 export default Component.extend({
-  classNames: ['SnapshotViewer'],
-
   snapshot: null,
+  classNames: ['SnapshotViewer'],
```
Comment (cadeParade) [inline app/components/snapshot-viewer.js]: P4: I usually put the class things above all the other things in a component

File: app/helpers/generate-fullscreen-snapshot-url.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import {helper} from '@ember/component/helper';
+
+export default helper(([snapshotId, selectedWidth, comparisonMode, isFullscreen]) => {
+  let location = window.location;
+  comparisonMode = comparisonMode || 'diff';
+  if (isFullscreen) {
+    return location.href;
+  } else {
+    return `${location.origin}${
```
Comment (cadeParade) [inline app/helpers/generate-fullscreen-snapshot-url.js]: lol ok prettier

File: app/components/snapshot-viewer.js (hunk 1 above)
Comment (cadeParade) [reply app/components/snapshot-viewer.js]: Oh, also this is what is causing the flash message to appear whenever you click anywhere on the snapshot.

**Comment by cadeParade** [comment]
Ok, so I was looking into this...the flash message is not appearing on the full screen view because of Z-INDEX.
Full screen view is in a modal, and that modal is on top of the flash message div. If you watch the "elements" panel in chrome inspector you can see that a flash message container does appear when you click the button, it is just hidden.

**Comment by PixelJanitor** [comment]
@cadeParade I made a test adjustment, but there are some additional failures I think you should look at. Otherwise, this is good to go! Let me know if I can help with anything else here.

File: app/components/snapshot-viewer.js (hunk 1 above)
Comment (cadeParade) [reply app/components/snapshot-viewer.js]: @PixelJanitor this is the line that was causing the tests to fail. I've removed it and now the tests should pass.

**Pull request merged**